tests/test_s3.py

```diff
@@ -124,20 +75,16 @@ def test_remove_object_exception():
         # Save the old external table minio client
         old_client = schema.external['share'].s3.client
 
-        # Apply our new minio client to the external table that has permissions restrictions
+        # Apply our new minio client which has a user that does not exist
         schema.external['share'].s3.client = minio_client
```

Comment:
Since `minio_client` is now only being called once, I wouldn't assign it to a variable but simply instantiate your client directly here. A bit easier to follow the logic in your test that way.